test/e2e/authorisation_test.go

```diff
@@ -552,38 +495,39 @@ var _ = framework.KubeDescribe("Authorization tests", func() {
 				},
 			},
 
-			//- poweruser can not use privileged PSP
-			{
-				msg: "access to use privileged PodSecurityPolicy for PowerUser should not be allowed",
-				reqBody: fmt.Sprintf(`{
-					"apiVersion": "authorization.k8s.io/v1beta1",
-					"kind": "SubjectAccessReview",
-					"spec": {
-					"resourceAttributes": {
-						"name": "privileged",
-						"namespace": "",
-						"verb": "use",
-						"group": "*",
-						"resource": "podsecuritypolicies"
-					},
-					"user": "sszuecs",
-					"group": [
-						"%s"
-					]
-					}
-				}`, powerUserGroup),
-				expect: expect{
-					status: http.StatusCreated,
-					body: fmt.Sprintf(`{
-					"apiVersion": "authorization.k8s.io/v1beta1",
-					"kind": "SubjectAccessReview",
-					"status": {
-						"allowed": false,
-						"reason":"unauthorized access sszuecs/[%s]"
-					}
-				}}`, powerUserGroup),
-				},
-			},
+			////- poweruser can not use privileged PSP
+			//// TODO: disable privileged PSP access for PowerUsers.
```

Comment:
why do we need to enable this?

Reply:
We still allow PowerUers to use privileged PSP in order to be able to create pods which uses sysctl commands e.g. for setting `vm.max_map_count`. Once we find away to allow this setting without privileged PSP (or remove the need for users to set this) we can disable privileged PSP for PowerUser.

Reply:
Some sysctl  values make sense to change for different workloads. Maybe we can enable users to only specify container/process based sysctl settings.

Reply:
I agree we can think of this, but I would really like to do this as part of another task to not increase the scope of this too much.

This change doesn't give more permissions than what already exists.

Reply:
fine for me